on-the-web/index.html

```diff
@@ -0,0 +1,89 @@
+---
+layout: default-no-anchor
+title: On the Web
+---          
+        <div class="callout large primary" id="bg-image-social" style="margin-bottom: 0rem;">
```

Comment:
Should I be seeing this background image in staging?

Reply:
it's the banner image behind the 'On the Web' heading

Reply:
![on-the-web](https://user-images.githubusercontent.com/2630707/30853285-40e9b5b6-a2a6-11e7-9a0e-2115e558548e.png)

Reply:
Huh, it *does* appear in Chromium. (The background images on other pages appear fine for me in Firefox.)

Reply:
Ah, I'd only looked at it on Safari, any ideas @lunson ?

Reply:
Hmm, I'm seeing a slight delay in loading for Firefox but it does work on 55.0.3

Reply:
Huh, yes, works fine for me in 55.0.3 from my laptop at home, failing only in the 52.3.0 here at work. Maybe just a weirdness that won't be a general problem.

Reply:
Maybe it's something odd with networking: whenever I look at the diff on GH then for this image I get,
![social](https://user-images.githubusercontent.com/2630707/30856855-9dcb2b56-a2b1-11e7-98a7-a7aeb33c5d8e.png)